docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -164,44 +165,57 @@ If the DUT is a controlee (e.g., light bulb), the TH spins a reference commissio
+If the DUT supports thread transport, an RCP dongle provisioned with a recommended RCP firmware for the default OTBR router that comes with the TH will be required to function properly. Currently, the OTBR can work with a Nordic RCP dongle or a SiLabs RCP dongle. Refer to <<ot-border-router-otbr-setup, Section 6, OT Border Router (OTBR) Setup>> on how to install the RCP firmware.
 
 ==== TH Installation on Raspberry Pi
 
-. Go to the https://drive.google.com/drive/u/1/folders/1ca7CG8QHHYy0g4ScSmiGL3KApNwtZ-w9[TH release location] and download the official TH image from the given link on the user’s PC/Mac.
 . Place the blank SD card into the user’s system USB slot. 
-. Open the https://www.raspberrypi.com/software/[Raspberry Pi Imager] or https://www.balena.io/etcher/[Balena Etcher] tool on the Mac/PC and select the image file from the drop-down list to flash.
-. After the SD card has been flashed with the image, remove the SD card and place it in the Raspberry Pi’s memory card slot.
+. Open the https://www.raspberrypi.com/software/[Raspberry Pi Imager] or https://www.balena.io/etcher/[Balena Etcher] tool on the Mac/PC and select the 'Ubuntu Server 22.04 LTS (64-bit)'.
+* Edit the SO custom settings to: 
+** username: ubuntu
+** password: raspberrypi
+** hostname: ubuntu
+* Make sure you have enabled the SSH service.
+. After the SD card has been flashed, remove the SD card and place it in the Raspberry Pi’s memory card slot.
 . Power on the Raspberry Pi and ensure that the local area network, display monitor and keyboard are connected.
-. Enter the default username and password:
-* username: ubuntu
-* password: raspberrypi
+. Enter the username and password.
+. Install the TH system:
+* Clone the TH repository: 
+** `$git clone -b <Target_Branch/Tag> https://github.com/project-chip/certification-tool.git`
+* Goto to TH folder: 
+** `$cd certification-tool`
+* Initialize the submodules: 
+** `$git submodule update --init --recursive`
+* Install/configure the TH dependencies: 
+** `$./scripts/pi-setup/auto-install.sh`
+** At the end of the script, select option 1 to restart the RaspberryPi.
+. Wait about 10 minutes. 
 . Using the _ifconfig_ command, obtain the IP address of the Raspberry Pi. The same IP address will be used to launch the TH user interface on the user's system using the browser.
-. Proceed with test configuration and execution (refer to <<test-configuration, Section 8, Test Configuration>> and <<test-case-execution, Section 9, Test Case Execution>> respectively).
+. Proceed with test configuration and execution (refer to <<test-configuration, Section 7, Test Configuration>> and <<test-case-execution, Section 8, Test Case Execution>> respectively).
```

Comment:
Should this be changed?

Reply:
??
<img width="430" alt="image" src="https://github.com/project-chip/certification-tool/assets/116589806/b7f63b71-e2ee-465c-9fca-1dab30285f56">